contracts/test/vault/vault.fork-test.js

```diff
@@ -0,0 +1,221 @@
+      await vault.rebase();
+    });
+  });
+
+  describe("Capital", () => {
+    it(`Shouldn't be paused`, async () => {
+      const { vault } = fixture;
+      expect(await vault.capitalPaused()).to.be.false;
+    });
+
+    it("Should allow to mint and redeem", async () => {
+      const { ousd, vault, josh, usdt } = fixture;
+      const balancePreMint = await ousd
+        .connect(josh)
+        .balanceOf(josh.getAddress());
+      await vault.connect(josh).mint(usdt.address, usdtUnits("50000"), 0);
+
+      const balancePostMint = await ousd
+        .connect(josh)
+        .balanceOf(josh.getAddress());
+      expect(balancePostMint.sub(balancePreMint)).to.approxEqual(
+        ousdUnits("50000")
+      );
+
+      await vault.connect(josh).redeem(ousdUnits("50000"), 0);
+
+      const balancePostRedeem = await ousd
+        .connect(josh)
+        .balanceOf(josh.getAddress());
+      expect(balancePreMint).to.approxEqual(balancePostRedeem);
+    });
+
+    it("Should have vault buffer disabled", async () => {
+      const { vault } = fixture;
+      expect(await vault.vaultBuffer()).to.equal("0");
+    });
+  });
+
+  describe("Oracle", () => {
+    it("Should have correct Price Oracle address set", async () => {
+      const { vault } = fixture;
+      expect(await vault.priceProvider()).to.equal(
+        "0x7533365d1b0D95380bc4e94D0bdEF5173E43f954"
+      );
+    });
+
+    it("Should return a price for minting with USDT", async () => {
+      const { vault, usdt } = fixture;
+      await vault.priceUSDMint(usdt.address);
+    });
+
+    it("Should return a price for minting with DAI", async () => {
+      const { vault, dai } = fixture;
+      await vault.priceUSDMint(dai.address);
+    });
+
+    it("Should return a price for minting with USDC", async () => {
+      const { vault, usdc } = fixture;
+      await vault.priceUSDMint(usdc.address);
+    });
+
+    it("Should return a price for redeem with USDT", async () => {
+      const { vault, usdt } = fixture;
+      const price = await vault.priceUSDRedeem(usdt.address);
+      expect(price).to.be.gte(utils.parseEther("1"));
+    });
+
+    it("Should return a price for redeem with DAI", async () => {
+      const { vault, dai } = fixture;
+      const price = await vault.priceUSDRedeem(dai.address);
+      expect(price).to.be.gte(utils.parseEther("1"));
+    });
+
+    it("Should return a price for redeem with USDC", async () => {
+      const { vault, usdc } = fixture;
+      const price = await vault.priceUSDRedeem(usdc.address);
+      expect(price).to.be.gte(utils.parseEther("1"));
+    });
+  });
+
+  describe("Assets & Strategies", () => {
+    it("Should NOT have any unknown assets", async () => {
+      const { vault } = fixture;
+      const assets = await vault.getAllAssets();
+
+      const knownAssets = [
+        // TODO: Update this every time a new asset is supported
+        "0xdAC17F958D2ee523a2206206994597C13D831ec7", // USDT
```

Comment:
these could also be read from addresses ( and others in this file as well)

Reply:
As per our discussion, leaving them as-is for now, (with a comment about it at the top of the file). Happy to change it anytime